transitgateway.tf

```diff
@@ -26,7 +26,37 @@ resource "aws_ec2_transit_gateway_vpc_attachment" "assessment" {
   vpc_id             = aws_vpc.assessment.id
 }
 
+# Break the association between the transit gateway VPC attachment and
+# the default transit gateway route table.
+#
+# It would be nice not to have to use the Terraform escape hatch for
+# this, but Terraform just doesn't support this sort of cross-account
+# foo right now.
```

Comment:
`foo` or `fu`?

Reply:
This is `foo` = `stuff`, not `fu` = `dark wizardry`.